src/org/scratchOrgSettingsGenerator.ts

```diff
@@ -326,30 +327,33 @@ export default class SettingsGenerator {
     objectsDir: string,
     allRecordTypes: string[],
     allbusinessProcesses: string[]
-  ) {
+  ): Promise<void> {
     if (this.objectSettingsData) {
```

Comment:
this and the one just below should be checked.  It's parallelization of async ops.